src/MudBlazor.Markdown/Extensions/EmphasisInlineEx.cs

```diff
@@ -6,7 +6,10 @@ internal static class EmphasisInlineEx
 {
 	public static bool TryGetEmphasisElement(this EmphasisInline emphasis, out string value)
 	{
-		const string italics = "i", bold = "b";
+		const string
+			italics = "i",
+			bold = "b",
+			strike = "del";
```

Comment:
Could you please put all three constants on the same line? And rename `strike` to `strikethrough`?

```cs
const string italics = "i", bold = "b", strikethrough = "del";
```